src/python/pants/backend/jvm/tasks/junit_run.py

```diff
@@ -90,10 +89,13 @@ def register_options(cls, register, register_jvm_tool):
                   'For example, 1/3 means run tests number 2, 5, 8, 11, ...')
     register('--suppress-output', action='store_true', default=True,
              help='Redirect test output to files in .pants.d/test/junit.')
-    # TODO(gmalmquist): Use the build root instead of the path of the first target by default.
     register('--cwd', default=_CWD_NOT_PRESENT, nargs='?',
-             help='Set the working directory. If no argument is passed, use the first target path. '
+             help='Set the working directory. If no argument is passed, use the build root. '
                   'If cwd is set on a target, it will supersede this argument.')
+    register('--cwd-default-to-first-target', action='store_true',
```

Comment:
We don't at Square; in fact I've updated pom-to-BUILD in my branch testing this to just appropriately set `cwd` for every junit_tests target, so we don't need it.

I found at least one integration test in pants that relied on that behavior though, so I thought we might have folks at other companies who depended on that behavior. If no one needs it I can just leave it out.